### DISCUSSION TITLE
Input 组件

### DISCUSSION THREAD

**Discussion by yangxiaolu1993** [jdf2e/nutui-react#13] [General]
1、限制输入长度：中文输入时，是按照字母而不是按照汉字，比如：长度限定为7，无法输入’中国‘

2、叉号出现后，失去焦点，虽然叉号消失，但是点击相应位置，清除功能依然生效

3、在demo中，基础用法中，输入的内容同步 

![image](https://user-images.githubusercontent.com/32215990/138284128-6ff6814c-2845-4dc8-bdbe-2a7a04424476.png)

4、prop：text-align 没效果
5、支持小数时，第一位输入是 . ，前面是否添加 0.

![image](https://user-images.githubusercontent.com/32215990/138284298-18a49103-079c-4965-9317-76bb42b4d8f6.png)

6、不能自定义 className/style

`<Input label="文本" className=‘custom’ defaultValue="禁止修改" disabled={true} />`

**Comment by zhenyulei** [comment]
基础用法中，输入上面的文本，下面的文本也会跟随变化
text-align 没有按照 nutui-vue 中实现该功能
没有失去焦点的事件
没有 clearable，展示清除 icon 功能
点击按钮修改默认值：输入框中变成了 value 黑色字体
点击按钮修改默认值：输入框中先输入文字，点击按钮后变成默认值，修改里面的内容，点击清除按钮后，再次点击按钮默认文案不在变化，也就是改变一次之后就不能在次改变了
自定义类型中的：整数、数字写asdasd，整数显示：e；数字写 .12213 显示：.12213

**Reply by vickyYE** [reply]
组件默认clearable，没有该功能设置成false则不会展示；
关于点击按钮修改默认值的两个问题没理解，例子中没有按钮点击操作
其他已修改

**Reply by vickyYE** [reply]
失去焦点事件在基础用法中异步里有，查看console可以看到事件返回

**Comment by vickyYE** [comment]
> 1、限制输入长度：中文输入时，是按照字母而不是按照汉字，比如：长度限定为7，无法输入’中国‘
> 
> 2、叉号出现后，失去焦点，虽然叉号消失，但是点击相应位置，清除功能依然生效
> 
> 3、在demo中，基础用法中，输入的内容同步
> 
> ![image](https://user-images.githubusercontent.com/32215990/138284128-6ff6814c-2845-4dc8-bdbe-2a7a04424476.png)
> 
> 4、prop：text-align 没效果 5、支持小数时，第一位输入是 . ，前面是否添加 0.
> 
> ![image](https://user-images.githubusercontent.com/32215990/138284298-18a49103-079c-4965-9317-76bb42b4d8f6.png)
> 
> 6、不能自定义 className/style
> 
> `<Input label="文本" className=‘custom’ defaultValue="禁止修改" disabled={true} />`

解答：
问题1，中文输入可以在限制中输入
问题5，保持与vue版同步未做修改
其他已修复
